### PR TITLE
Add some options commands to discord

### PR DESCRIPTION
**Pull request by jmaslek**
Just looking at calls/puts and open interest.

### CODE REVIEW

File: discordbot/stocks/options/calls.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                str_start += 4000
+                i += 1
+
+            await pagination(columns, ctx)
+
+        await ctx.send(embed=embed)
+
+    except Exception as e:
+        embed = discord.Embed(
+            title="ERROR Stocks-Options: Expirations",
```
Comment (northern-64bit) [inline discordbot/stocks/options/calls.py:66]: I would write Stock-Options here.

File: discordbot/stocks/options/expirations.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        embed.set_author(
+            name=cfg.AUTHOR_NAME,
+            icon_url=cfg.AUTHOR_ICON_URL,
+        )
+
+        await ctx.send(embed=embed)
+
+    except Exception as e:
+        embed = discord.Embed(
+            title="ERROR Stocks-Options: Expirations",
```
Comment (northern-64bit) [inline discordbot/stocks/options/expirations.py:36]: Also Stock-Options here.

File: discordbot/stocks/options/calls.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+
+async def calls_command(ctx, ticker: str = "", expiration_date: str = ""):
+    """Show calls for given ticker and expiration"""
+    try:
+        # Debug
+        if cfg.DEBUG:
+            print(f"!stocks.opt.calls {ticker.upper()} {expiration_date}")
+
+            # Check for argument
```
Comment (northern-64bit) [inline discordbot/stocks/options/calls.py:16]: Take away one tab here

File: discordbot/stocks/options/options_menu.py [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+            f"1️⃣ !stocks.opt.calls {ticker} <EXPIRATION> \n"
+        )
+
+        cols_temp.append(text)
+        for col in cols_temp:
+            cols.append(
+                discord.Embed(
+                    description=col,
+                    colour=cfg.COLOR,
+                    title="Stocks: Technical Analysis (TA) Menu",
```
Comment (northern-64bit) [inline discordbot/stocks/options/options_menu.py:109]: I think that the title is wrong here.

File: discordbot/stocks/options/puts.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                str_start += 4000
+                i += 1
+
+            await pagination(columns, ctx)
+
+        await ctx.send(embed=embed)
+
+    except Exception as e:
+        embed = discord.Embed(
+            title="ERROR Stocks-Options: Expirations",
```
Comment (northern-64bit) [inline discordbot/stocks/options/puts.py:66]: Stock-Options should be here to.

**Review by northern-64bit — COMMENTED** [review]
Everything looks great. There are only some small things to improve. The only criticism I can give is that there are some inconsistency at the message titles (Is it "Stocks:" or "Stock-options:" or "Options"?). Great job!!!

**Comment by jmaslek** [comment]
> Everything looks great. There are only some small things to improve. The only criticism I can give is that there are some inconsistency at the message titles (Is it "Stocks:" or "Stock-options:" or "Options"?). Great job with the pr!!!

Fixed everything to the Stock-Options as you suggested

**Review by northern-64bit — APPROVED** [review]

**Pull request merged**